tests/cli/log2timeline_tool.py

```diff
@@ -93,19 +95,21 @@ class Log2TimelineToolTest(test_lib.CLIToolTestCase):
                         system CPUs minus one].
 """)
 
-  def _CheckOutput(self, output, expected_output):
+  def _CheckOutput(self, output, expected_output, encoding='utf-8'):
     """Compares the output against the expected output.
 
     The actual processing time is ignored, since it can vary.
 
     Args:
       output (bytes): tool output.
-      expected_output (list[bytes]): expected tool output.
+      expected_output (list[str]): expected tool output.
+      encoding (optional[str]): encoding of the output.
     """
-    output = output.split(b'\n')
+    output = codecs.decode(output, encoding)
```

Comment:
why decode the output instead of encoding the expected value?

Don't we want to check the function output as much as possible unaltered?

Reply:
I don't think this matters much, and this reduces the amount of boilerplate. When I fix up all the CLI tests, I'll likely add a ReadOutputString() or similar, as for the output tests.

The OutputWriters all write strings, so thing using the TestOutputWriters should also care about strings, not bytes.